elixir/.env.example

```diff
@@ -0,0 +1,7 @@
+SCYLLADB_USERNAME=
+SCYLLADB_PASSWORD=
+SCYLLADB_NODE_1=
+SCYLLADB_NODE_2=
+SCYLLADB_NODE_3=
+SCYLLADB_KEYSPACE=
+SCYLLADB_TABLE=
```

Comment:
add a last line in file

Reply:
Thanks! Done https://github.com/scylladb/scylla-cloud-getting-started/pull/22/commits/fb9812b16ed8af1b1fd79d46394cb65cf658dc42